tests/sflow/test_sflow.py

```diff
@@ -109,6 +109,28 @@ def config_dut_ports(duthost, ports, vlan):
     config_reload(duthost, config_source='config_db', wait=120)
     time.sleep(5)
 
+# ----------------------------------------------------------------------------------$
+
+
+def get_default_agent(duthost):
+    # get_default_agent function is used to get the agentip selected by the hsflowd dameon.
```

Comment:
Can you help to change the typo, when i cherry-pick to our local, it failed in our spell checker
dameon -> daemon

Reply:
I think we also need this fix in the 202205 branch right?

Reply:
well done. nice catch @nhe-NV

Reply:
Addressed the changes. Kindly merge this PR in which are all the branches https://github.com/sonic-net/sonic-mgmt/pull/9766 is merged, as this is the deterministric/alternative approach for https://github.com/sonic-net/sonic-mgmt/pull/9766

Reply:
once pr checkers are passing i will.
as for cherry pick to other branches, it will be followed as the label was set.